itest/rust/build.rs

```diff
@@ -148,8 +148,9 @@ fn collect_inputs() -> Vec<Input> {
     push!(inputs; PackedStringArray, PackedStringArray, PackedStringArray(), PackedStringArray::new());
     push!(inputs; PackedVector2Array, PackedVector2Array, PackedVector2Array(), PackedVector2Array::new());
     push!(inputs; PackedVector3Array, PackedVector3Array, PackedVector3Array(), PackedVector3Array::new());
-    #[cfg(since_api = "4.3")]
-    push!(inputs; PackedVector4Array, PackedVector4Array, PackedVector4Array(), PackedVector4Array::new());
+    if godot_bindings::since_api("4.3") {
+        push!(inputs; PackedVector4Array, PackedVector4Array, PackedVector4Array(), PackedVector4Array::new());
+    }
```

Comment:
Could you elaborate this change?

Reply:
this is a build script, so `#[cfg(since_api = "4.3")]` doesn't do anything.

Reply:
Oh, makes sense. Strange that this didn't get caught by the 4.1/4.2 compat checks :thinking:

Reply:
Thanks for the extra comment!